src/library.js

```diff
@@ -209,6 +209,8 @@ LibraryManager.library = {
 #endif
     }
 
+    let alignUp = (x, multiple) => x + (multiple - x % multiple) % multiple;
```

Comment:
Although its probably fine we haven't actually done any `var` -> `let` transitioning yet.. so this would be first `let` in the generated code... maybe OK but to be safe you might want to stick with `var` to avoid any unrelated regressions being bisected to this PR.